src/app/App.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react"
+import reactLogo from "../assets/react.svg"
+import "./App.css"
+import i18nManager from "./i18n"
+import Layout from "./providers/layout"
+import ReactQuery from "./providers/react-query"
+import Redux from "./providers/state"
```

Comment:
Please organize the import's strategy.
For example, import { } from "react" goes first. Then go imports from npm dependencies. Then goes a section with absolute imports. Then goes a section with relative imports.  The last ones are css related.
Each section should be divided by an empty line;

Example:
<img width="472" alt="image" src="https://user-images.githubusercontent.com/20073193/203064005-d79dfadb-4143-4ca7-8ddb-46ec2640ae67.png">

You can choose whatever strategy you like but stick to one in every file.

Reply:
Done